sites/upsun/src/get-started/stacks/magento.md

```diff
@@ -0,0 +1,192 @@
+applications:
+    app:
+        # The runtime the application uses.
+        type: php:8.3
+        # Specify additional PHP extensions that should be loaded.
+        runtime:
+            extensions:
+                - xsl
+                - sodium
+                - redis
+                - blackfire
+        variables:
+            env:
+                NVM_VERSION: master
+                NODE_VERSION: 20
+                MAGENTO_DC_INDEXER__USE_APPLICATION_LOCK: true
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOGINVENTORY_STOCK__SIMPLE: 200
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOG_CATEGORY_PRODUCT: 666
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOGSEARCH_FULLTEXT__PARTIAL_REINDEX: 100
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOGSEARCH_FULLTEXT__MYSQL_GET: 500
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOGSEARCH_FULLTEXT__ELASTIC_SAVE: 500
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOG_PRODUCT_PRICE__SIMPLE: 200
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOG_PRODUCT_PRICE__DEFAULT: 500
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOG_PRODUCT_PRICE__CONFIGURABLE: 666
+                MAGENTO_INDEXER_BATCH_SIZE__CATALOGPERMISSIONS_CATEGORY: 999
+                MAGENTO_INDEXER_BATCH_SIZE__INVENTORY__SIMPLE: 210
+                MAGENTO_INDEXER_BATCH_SIZE__INVENTORY__DEFAULT: 510
+                MAGENTO_INDEXER_BATCH_SIZE__INVENTORY__CONFIGURABLE: 616
+            php:
+                memory_limit: "512M"
+              ...
+```
+
+## Create project
+
+Copy the following command into command line:
+
+`upsun project:create --init-repo https://github.com/platformsh-templates/magentoCE24/`
+
+In command line, you will be prompted to do the following:
+
+- Select an organization to create the project under
+- Select a title for your project 
+- Select a [region to deploy from](https://docs.upsun.com/development/regions.html#regions)
+- Select your default branch
+- Select whether you would like to set your new project as the remote for any existing repositories that have been detected under your organization
+
+Once you have made your selections, the Upsun bot will activate your project.
+
+{{< note title="Note" theme="info" >}}
+Note that this project will not have an estimated monthly cost.
+
+{{< /note >}}
+
+When your project is created, you will be provided with the following details to access it:
+
+- `Region`
+- `Project ID`
+- `Project title`
+- `Project URL`
+- `Git URL` 
+
+## Configure resources
+
+Copy the Project URL into your browser. You should see your newly created project in the Upsun console. For example, if you had named your Magento project `Mage`, you would see something similar to the screenshot below:
+
+![Your magento project in the Upsun console](/images/guides/magento/mage-console-1.png) 
+
+You will be prompted to configure your resources. At this stage you can select the CPU, RAM, instances and disk size for your Magento project. 
```

Comment:
see DM irt to convo with Jared about container_profiles and minimum resource amounts.